### PR TITLE
Remove RUN_SERIAL TRUE test=develop

### PR DESCRIPTION
**Pull request by chalsliu**
### PR types
Others

### PR changes

Others 
### Describe

生效部分是`RUN_TYPE=EXCLUSIVE`，非生效部分是`RUN_SERIAL TRUE`，把非生效部分去掉

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/CMakeLists.txt [hunk 1]
```diff
@@ -414,13 +414,13 @@ set_tests_properties(test_parallel_executor_crf test_sync_batch_norm_op test_inp
...
         test_parallel_executor_seresnext_with_fuse_all_reduce_gpu
-        PROPERTIES LABELS "RUN_TYPE=DIST" RUN_SERIAL TRUE)
+        PROPERTIES LABELS "RUN_TYPE=DIST")
 
 if(NOT WIN32 AND NOT APPLE)
-    set_tests_properties(test_imperative_data_loader_base PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
-    set_tests_properties(test_imperative_data_loader_fds_clear PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
+    set_tests_properties(test_imperative_data_loader_base PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    set_tests_properties(test_imperative_data_loader_fds_clear PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
     # set_tests_properties(test_imperative_data_loader_exception PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/CMakeLists.txt]: 422行注释部分中的`RUN_SERIAL`也可以去掉

**Comment by chalsliu** [comment]
修改前后结果一致：
https://xly.bce.baidu.com/paddlepaddle/paddle/ipipe/#/detail/1141420/job/1624031
1 card TestCases count is 128
1 card TestCases count is 861
2 card TestCases count is 30
exclusive TestCases count is 43

https://xly.bce.baidu.com/paddlepaddle/paddle/ipipe/#/detail/1139243/job/1621042
1 card TestCases count is 128
1 card TestCases count is 861
2 card TestCases count is 30
exclusive TestCases count is 43

File: python/paddle/fluid/tests/unittests/CMakeLists.txt (hunk 1 above)
Comment (chalsliu) [reply python/paddle/fluid/tests/unittests/CMakeLists.txt]: done

**Review by luotao1 — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/CMakeLists.txt [hunk 2]
```diff
@@ -414,13 +414,13 @@ set_tests_properties(test_parallel_executor_crf test_sync_batch_norm_op test_inp
...
-    # set_tests_properties(test_imperative_data_loader_exception PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
-    set_tests_properties(test_multiprocess_dataloader_static PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
-    set_tests_properties(test_multiprocess_dataloader_dynamic PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
-    set_tests_properties(test_multiprocess_dataloader_exception PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE" RUN_SERIAL TRUE)
+    set_tests_properties(test_imperative_data_loader_base PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    set_tests_properties(test_imperative_data_loader_fds_clear PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    # set_tests_properties(test_imperative_data_loader_exception PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    set_tests_properties(test_multiprocess_dataloader_static PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    set_tests_properties(test_multiprocess_dataloader_dynamic PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
+    set_tests_properties(test_multiprocess_dataloader_exception PROPERTIES LABELS "RUN_TYPE=EXCLUSIVE")
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/CMakeLists.txt:425]: python/paddle/fluid/tests/unittests/CMakeLists.txt中还有13处`SERIAL`，都可以进行删除

Comment (chalsliu) [reply python/paddle/fluid/tests/unittests/CMakeLists.txt:425]: ok

**Pull request merged**